src/crab/ebpf_domain.hpp

```
@@ -90,30 +112,44 @@ class EbpfDomain final {
     [[nodiscard]]
-    Interval get_map_max_entries(const Reg& map_fd_reg) const;
+    Interval get_map_max_entries(const Reg& map_fd_reg, const ebpf_platform_t& platform) const;
 
     bool get_map_fd_range(const Reg& map_fd_reg, int32_t* start_fd, int32_t* end_fd) const;
 
   private:
-    // private generic domain functions
+    // Generic domain mutators. All maintain the
+    // `state.is_bottom() <=> !stack` invariant via normalize_after_state_mutation.
     void add_value_constraint(const LinearConstraint& cst);
+    void assume_eq_types(Variable v1, Variable v2);
```

Clang (14.0.6)

[warning] parameter name 'v1' is too short, expected at least 3 characters

(readability-identifier-length)